patterns/2-structured/maturity-model.md

```diff
@@ -153,60 +134,55 @@ One of the possible reasons for introducing InnerSource into organisations can b
+### Rewards*To drive adoption, extrinsic motivators can be used to increase cross-team collaboration.
 
 * RW-0: No rewards.
 * RW-1: Leaders are encouraged to reward exceptional collaborations, but there are no policies or processes established.
 * RW-2: Standard processes are established to reward collaborations outside the developers' teams. Team leaders or boards decide who has to be rewarded.
-* RW-3: Rewards are not only proposed by the organization, but the communities are able to define their more valuable rewards. The community is responsible to decide who has to be rewarded.
+* RW-3: Rewards are not only proposed by the organization, but the communities can define their more valuable rewards. The community is responsible to decide who has to be rewarded.
 
-**Monitoring Policies**
+#### Monitoring Policies
 
-InnerSource projects need a means for self assessment. Metrics can be one aspect to facilitate this assessment. Also, in organisations with a mature InnerSource adoption level we expect adoption of the method to be tracked based on clear, agreed upon metrics.
+InnerSource projects need a means for self-assessment. Metrics can be one aspect to facilitate this assessment. Also, in organizations with a mature InnerSource adoption level, we expect the adoption of the method to be tracked based on clear, agreed-upon metrics.
 
 * MP-0: No existing monitoring policies at any level in the organization.
 * MP-1: Metrics are important for certain teams, and they start using them in an isolated way.
-* MP-2: There is a strategy at the organizational level with respect to metrics that help to validate specific policies across the organization. This monitoring policy exists at the level of some InnerSource projects.
-* MP-3: There are clear guidelines, recommendations, and trainings about the use of metrics with certain infrastructure provided by the organization. This works at both levels: InnerSource program to understand the general InnerSource adoption within the organization, and at the level of InnerSource projects.
+* MP-2: There is a strategy at the organizational level concerning metrics that help to validate specific policies across the organization. This monitoring policy exists at the level of some InnerSource projects.
+* MP-3: There are clear guidelines, recommendations, and training about the use of metrics with certain infrastructures provided by the organization. This works at both levels: InnerSource program to understand the general InnerSource adoption within the organization and at the level of InnerSource projects.
 
-**Support and Maintenance**
+#### Support and Maintenance
 
-Not only should feature development be owned by InnerSource teams - support and maintenance is also part of the teams core tasks.
+Not only should feature development be owned by InnerSource teams - support and maintenance is also part of the team's core tasks.
```

Comment:
```suggestion
Not only should feature development be owned by InnerSource teams - support and maintenance is also part of the teams' core tasks.
```

The sentence starts with "teams" as plural. So maybe it has to be the genitive of the plural form here?